src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
+		None = 0,
+		PowerDidChange = 1,
+		SSIDDidChange = 2,
+		BSSIDDidChange = 3,
+		CountryCodeDidChange = 4,
+		LinkDidChange = 5,
+		LinkQualityDidChange = 6,
+		ModeDidChange = 7,
+		ScanCacheUpdated = 8,
+		VirtualInterfaceStateChanged = 9,
+		RangingReportEvent = 10,
+		Unknown = int.MaxValue
```

Comment:
@chamons we use the 64-bits value (e.g. `src/MediaPlayer/MediaPlayer.cs`) as our enums are always 64 bits (in unified) but it's worth a test. to make sure the value is correct under 32bits (at least for classic where the underlying type is changed).